File: exs/mundo_1/java/014.java
// Conversor de temperaturas: escreva um programa que converta uma temperatura
// digitada em ºC para ºF
import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        System.out.print("Celsius: ");
        float celsius = scan.nextFloat();
        String fahrenheit = String.format("%.2f", (celsius * 9/5) + 32);
        System.out.println(fahrenheit+"°F");
    }
}

File: exs/mundo_1/c/014.c
/*
    014 - Conversor de temperaturas
        Conversor de temperaturas: escreva um programa que converta uma temperatura
        digitada em ºC para ºF
*/

#include <stdio.h>

int main(void){
    float temp_graus;
    printf("Digite a temperatura em graus a ser convertida: ");
    scanf("%f", &temp_graus);

    printf("Conversão para Fahrenheit: %.1f°F\n", temp_graus * 1.8 + 32);
    return 0;
}
